src/EventHandler/Message/GroupMessage.php

```diff
@@ -352,11 +352,12 @@ public function disableTopics(): void
      * Create a [forum topic](https://core.telegram.org/api/forum); requires [`manage_topics` rights](https://core.telegram.org/api/rights).
      *
      * @param string $title Topic title (maximum UTF-8 length: 128)
-     * @param integer $icon ID of the [custom emoji](https://core.telegram.org/api/custom-emoji) used as topic icon. [Telegram Premium](https://core.telegram.org/api/premium) users can use any custom emoji, other users can only use the custom emojis contained in the [inputStickerSetEmojiDefaultTopicIcons](https://docs.madelineproto.xyz/API_docs/constructors/inputStickerSetEmojiDefaultTopicIcons.html) emoji pack.
-     * @param IconColor $color If no custom emoji icon is specified, specifies the color of the fallback topic icon (RGB)
+     * @param IconColor|int $icon ID of the [custom emoji](https://core.telegram.org/api/custom-emoji) used as topic icon.
+     * [Telegram Premium](https://core.telegram.org/api/premium) users can use any custom emoji, other users can only use the custom emojis contained in the [inputStickerSetEmojiDefaultTopicIcons](https://docs.madelineproto.xyz/API_docs/constructors/inputStickerSetEmojiDefaultTopicIcons.html) emoji pack.
+     * If no custom emoji icon is specified, specifies the color of the fallback topic icon (RGB)
      * @throws InvalidArgumentException
      */
-    public function createTopic(string $title, int $icon, IconColor $color = IconColor::NONE): DialogTopicCreated
+    public function createTopic(string $title, IconColor|int $icon = IconColor::NONE): DialogTopicCreated
```

Comment:
Just IconColor is OK, I think, no other values are allowed server-side AFAIK

Reply:
icon color or emoji !